frontend/src/components/common/header/header.tsx

```diff
@@ -23,9 +23,9 @@ const Header: React.FC = () => {
         {hasUser ? (
           <>
             <ul className={styles.navigation}>
-              <li className={styles.liNavigation}>Podcasts</li>
-              <li className={styles.liNavigation}>Streaming</li>
-              <li className={styles.liNavigation}>Setting</li>
+              <Link to={AppRoute.ROOT}>
+                <li className={styles.liNavigation}>Podcasts</li>
+              </Link>
```

Comment:
```suggestion
             
                <li className={styles.liNavigation}>
                 <Link to={AppRoute.ROOT}>
                   Podcasts
                  /Link>
                 </li>
              
```
`ul` element can include only `li` elements